src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -201,6 +203,52 @@
 //%DllMap%
 }
 
+int32_t GlobalizationNative_LoadICUData(void * pData);
+
+static int32_t load_icu_data()
+{
+    char path [1024];
+    int res;
+    
+    const char *dname = "icudt.dat";
+    const char *bundle = get_bundle_path ();
+
+    os_log_info (OS_LOG_DEFAULT, "Loading ICU data file '%s'.", dname);
+    res = snprintf (path, sizeof (path) - 1, "%s/%s", bundle, dname);
+    assert (res > 0);
+   
+    char *icu_data = NULL;
+    FILE *fp = fopen(path, "rb");
+    if (fp != NULL) {
```

Comment:
so if the file doesn't exists we call `GlobalizationNative_LoadICUData` with `null` here?

Reply:
yeah, we'll need to do something about it.